Carthage.md

```diff
@@ -57,8 +57,8 @@ binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseStorageBinary.jso
 - Use Finder to open `Carthage/Build/iOS`.
 - Copy the contents into the top level of your Xcode project and make sure
     they're added to the right build target(s).
-- Add the -ObjC flag to "Other Linker Flags".
-- Make sure that the build target(s) includes your project's `GoogleService-Info.plist`.
+- Add `$(OTHER_LDFLAGS) -ObjC` flag to "Other Linker Flags" in "Build Settings".
```

Comment:
What circumstances is $(OTHER_LDFLAGS) needed here? Isn't this the field that sets that value? Should it be `$inherited` instead?

Reply:
I imagine it to be the same thing. 
I just followed this Apple doc https://developer.apple.com/library/archive/qa/qa1490/_index.html